src/Workspaces/CSharp/Portable/Utilities/UsingsAndExternAliasesOrganizer.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.CodeAnalysis.CSharp.Utilities
 {
     internal static partial class UsingsAndExternAliasesOrganizer
     {
-        private static readonly SyntaxTrivia s_newLine = SyntaxFactory.ElasticCarriageReturnLineFeed;
+        private static readonly SyntaxTrivia s_newLine = SyntaxFactory.CarriageReturnLineFeed;
```

Comment:
does VB have the same problem?

Reply:
hmm, It seems that we don't have an `ICodeCleanupFixerProvider` for VisualBasic..